example/traffic_mgmt/odp_traffic_mgmt.c

```diff
@@ -293,6 +293,40 @@ static uint8_t  g_print_tm_stats   = TRUE;
 
 static void tester_egress_fcn(odp_packet_t odp_pkt);
 
+static uint64_t tm_shaper_min_rate;
+static uint64_t tm_shaper_max_rate;
+static uint32_t tm_shaper_min_burst;
+static uint32_t tm_shaper_max_burst;
+
+static uint64_t
+clamp_rate(uint64_t rate)
+{
+	uint64_t val = MIN(MAX(rate, tm_shaper_min_rate), tm_shaper_max_rate);
+
+	/* PIR can be zero just with CIR valid and vice versa */
+	if (!rate)
+		return 0;
+
+	if (val != rate)
+		printf("INFO: Clamped shaper rate from %lu bps to %lu bps\n",
```

Comment:
The format specifiers in this and other printfs do not match the data types, which causes CI tests to fail. When the argument type is uint32_t, use PRIu32 and when the type is uint64_t, use PRIu64.